data/assets/educational/scale/eiffeltower.asset

```diff
@@ -0,0 +1,84 @@
+local earthAsset = asset.require('scene/solarsystem/planets/earth/earth')
+local sunAsset = asset.require('scene/solarsystem/sun/sun')
+
+local modelFolder = asset.syncedResource({
+	Name = "Eiffel Tower Model",
+	Type = "HttpSynchronization",
+	Identifier = "eiffel_tower_model",
+	Version = 1
+  })
+  
+local eiffelTower = {
+	Identifier = "eiffelTower",
+	Parent = earthAsset.Earth.Identifier,
+	Renderable = {
+		Type = "RenderableModel",
+		GeometryFile = modelFolder .. "eiffeltower.osmodel",
+		ModelScale = "Centimeter",
+		RotationVector = {0,45,0},
+		LightSources = {
+			{
+				Type = "SceneGraphLightSource",
+				Identifier = "sunLightSourceForEiffelTower",
+				Node = sunAsset.Sun.Identifier
+			},
+		}
+	},
+	GUI = {
+		Name = "Eiffel Tower",
+		Path = "/Scale Objects"
+	},
+	Transform = {
+		Translation = {
+			Type = "GlobeTranslation",
+			Globe = earthAsset.Earth.Identifier,
+			Longitude = 2.29448,
+			Latitude = 48.85824,
+			Altitude = 0,
+			UseHeightmap = true
+		},
+		Rotation = {
+			Type = "GlobeRotation",
+			Globe = earthAsset.Earth.Identifier,
+			Longitude = 2.29448,
+			Latitude = 48.85824,
+			UseHeightmap = false
+		},
+		Scale = {
+			Type = "StaticScale",
+			Scale = 4.38
+		}
+	}
+}
+
+local updatePositionAction = {
+    Identifier = "dropEiffelTower",
```

Comment:
I "namespaced" all of my actions with `os.` in the beginning to mean that they are defined by our team, so I'd suggest the same here:  `os.drop_eiffel_tower` or `os.dropeiffeltower`